level1/bmi.rb

```diff
@@ -0,0 +1,18 @@
+class Bmi
+
+  @@cant_decimals = 1
```

Comment:
It's not necessary a variable of class.
```suggestion
  cant_decimals = 1
```